src/components/HeaderBar.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactComponent as KHLogo } from '../assets/logo.svg';
+import { ReactComponent as DrawerHamburger } from '../assets/drawerHamburger.svg';
+import { useNavigate } from 'react-router-dom';
+
+export default function HeaderBar() {
+	let navigate = useNavigate();
+
+	return (
+		<div className="flex justify-between w-full p-4 shadow-headerbar mb-3">
+			<div className="flex items-center">
+				<button
+					onClick={() => {
+						/*TODO*/
+					}}
+				>
+					<DrawerHamburger className="mr-4" />
+				</button>
+
+				<button onClick={() => navigate('/')}>
```

Comment:
Any particular reason you're using buttons with `navigate()` instead of[ `<Link to="/some-path" />`](https://github.com/remix-run/react-router/blob/main/docs/getting-started/tutorial.md#listing-the-invoices)?